sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileClient.java

```diff
@@ -1101,7 +1101,7 @@ public DataLakeFileOpenInputStreamResult openInputStream(DataLakeFileInputStream
         BlobInputStreamOptions convertedOptions = Transforms.toBlobInputStreamOptions(options);
         BlobInputStream inputStream = blockBlobClient.openInputStream(convertedOptions);
         return new InternalDataLakeFileOpenInputStreamResult(inputStream,
-            Transforms.toPathProperties(inputStream.getProperties()));
+            Transforms.toPathProperties(inputStream.getProperties(), null));
```

Comment:
Do we need to add encryption context as an option for this API?

Reply:
Not possible to extract encryption context from blockblob here, so we'll skip

Reply:
Are we able to access the raw response from the `GetBlob()` request?  If so, we can extract encryption context from the response headers.

Reply:
Checked responses that we get from BlobInputStream in Fiddler - `x-ms-encryption-context` is not present in the response headers. Only times `x-ms-encryption-context` is present in the response headers are for the create() method, read() method, and getProperties() method (same as .NET).